fastlane/Fastfile

```diff
@@ -95,7 +95,9 @@ lane :execute_tests do
 
     # Run the tests
     #
-    sh("bundle exec rake test_all")
+    # sh("set DEBUG=1")
```

Comment:
Have you tried `export DEBUG=1`? 😇

Reply:
That doesn't work on Windows. The last commit does trigger a debug run of `rspec`, but it still doesn't help me find out why do [these](https://github.com/fastlane/fastlane/pull/17412/files#diff-6f52cc9448fdd93818fee689b1d7af68R24-R55) fail:

```
[00:05:15] [08:55:28]: ▸ [08:55:28]: Cloning remote git repo...
[00:05:15] [08:55:28]: ▸ [08:55:28]: $ git clone C:/Users/appveyor/AppData/Local/Temp/1/fl_spec_import_from_git_source20201009-1424-lrgxrf C:/Users/appveyor/AppData/Local/Temp/1/fl_spec_import_from_git_cache20201009-1424-19splbk/fl_spec_import_from_git_source20201009-1424-lrgxrf  -n 
[00:05:15] [08:55:28]: [08:55:28]: ▸ Cloning into 'C:/Users/appveyor/AppData/Local/Temp/1/fl_spec_import_from_git_cache20201009-1424-19splbk/fl_spec_import_from_git_source20201009-1424-lrgxrf'...
[00:05:15] [08:55:28]: [08:55:28]: ▸ warning: You appear to have cloned an empty repository.
[00:05:15] [08:55:28]: [08:55:28]: ▸ done.
```

Reply:
Any of that git code runs properly?

Reply:
I don't understand your question. On Linux, it was failing because git email and name weren't configured, and the commit simply failed. On Windows, it looks like it's the same problem: the commits are failing, but I don't know why.

Reply:
Could you use Open3 to spawn the command and recover stdout, stderr and exit code, maybe that helps you gathering info why isn't working on Windows.

Reply:
Instead of the backticks?

Reply:
https://docs.ruby-lang.org/en/2.0.0/Open3.html this is the most low level in order to spawn a process that I know.